lychee-lib/src/filter/mod.rs

```diff
@@ -138,6 +140,13 @@ impl Filter {
         uri.is_mail() && !self.include_mail
     }
 
+    #[inline]
+    #[must_use]
+    /// Whether tel aren't checked (which is the default)
+    pub fn is_tel_excluded(&self, uri: &Uri) -> bool {
+        uri.is_tel() && !self.include_tel
+    }
+
```

Comment:
Can we remove that, as there's no way to check phone numbers?
I've updated the code below as well.

```suggestion

```